content/docs/secure/encryption/rekey.md

```diff
@@ -0,0 +1,177 @@
+
+```yaml
+# Rekey controller image name.
+image: dellemc/csm-encryption-rekey-controller:v0.1.0
+
+# Rekey controller image pull policy.
+# Allowed values:
+#  Always: Always pull the image.
+#  IfNotPresent: Only pull the image if it does not already exist on the node.
+#  Never: Never pull the image.
+imagePullPolicy: IfNotPresent
+
+# This value is required and must match encryption.pluginName value
+# of the corresponding Dell CSI driver.
+provisioner: sec-isilon.dellemc.com
+# This represents the port number used by encryption driver Rest API server.
+port: 3838
+```
+> **NOTE**: It is important that the `provisioner` reflect the same name as the `pluginName` specified in the standard encryption values.yaml file, and the `port`
+reflect the same port number as the encryption driver REST API server would be listening on. If they don't match then all rekey attempts will fail.
+
+## Deploy Rekey Controller
+
+Copy the values.yaml to a local file. Once the CSM encryption Rekey Controller local values.yaml file has been
+adjusted for the current cluster, deploy the controller by installing
+the helm chart. As an example:
+
+``` helm install --values local-values.yaml isilon-rekey csm-encryption-rekey-controller```
+
+A rekey-controller pod should now be up and running.
+
+
+## Rekey Controller Usage
+
+The CSM Encryption Rekey Controller must be installed on a Kubernetes cluster in order to be able to Rekey encrypted 
+volumes. The general procedure for rekeying is to create a rekey custom resource via a simple yaml configuration. This 
+will kick off a rekey process on the PV specified as the `volume` in the resource. 
+
+## Rekey with dellctl
+
+If dellctl CLI is installed, rekey'ing a volume is simple. First, identify a volume with a PV that is encrypted with the CSM encryption driver
+provisioner. Verify that there is at least one volume attachment for this PV.
+
+For example, let's rekey an encrypted PV with the name ``K8s-asdf1234``:
+
+``dellctl encryption rekey myrekey k8s-asdf1234``
+
+```
+INFO rekey request "myrekey2" submitted successfully for persistent volume "k8s-112a5d41bc".
+INFO Run 'dellctl encryption rekey-status myrekey2' for more details.
+```
+
+Then to check the status of the newly created rekey with the name ``myrekey``:
+
+``dellctl encryption rekey-status myrekey``
+
+```INFO Status of rekey request myrekey2 = completed```
+
+See below for possible Status values and explanations.
+
+## Rekey with manually created Rekey CRs
+
+Identify a volume with a PV that is encrypted with the CSM encryption driver 
+provisioner. Verify that there is at least one volume attachment for this PV. 
+
+Now create a Rekey Custom Resource to start the rekey of a volume. For example, there is a PV with a name
+`k8s-09a76734f` as follows:
+```
+Name:            k8s-029a76734f
+Labels:          <none>
+Annotations:     pv.kubernetes.io/provisioned-by: sec-isilon.dellemc.com
+...
+Events:                <none>
+```

```

Comment:
Why is this needed?

Reply:
This was to convey the pv name. Feels redundant in this context, removed.